setup.py

```diff
@@ -18,7 +18,7 @@
     author="Yoshiki Vazquez-Baeza",
     author_email="yoshiki@ucsd.edu",
     description="Display ordination plots",
-    license="bsd",
+    license='BSD-3-Clause',
```

Comment:
In this case we are basing it on the [SPDX ID](https://spdx.org/licenses/)
which `npm` lints for.

-Evan

On Tue, Jul 19, 2016 at 4:24 PM, Yoshiki Vázquez Baeza <
notifications@github.com> wrote:

> In setup.py
> <https://github.com/qiime2/q2-emperor/pull/8#discussion_r71442247>:
>
> > @@ -18,7 +18,7 @@
> >      author="Yoshiki Vazquez-Baeza",
> >      author_email="yoshiki@ucsd.edu",
> >      description="Display ordination plots",
> > -    license="bsd",
> > +    license='BSD-3-Clause',
>
> I think the way to write this (from our interactions with the conda-forge
> team) is BSD Clause-3.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/qiime2/q2-emperor/pull/8/files/d569ddf053ba9da3e75066a080eafc6e1178fd3f#r71442247>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADyuZNpGxgSs8D-sZ59Em9qaf5hRmLJvks5qXVy1gaJpZM4JQR5b>
> .
>